specifyweb/frontend/js_src/lib/components/Statistics/Categories.tsx

```diff
@@ -292,6 +297,28 @@ export function Categories({
           {statsText.addACategory()}
         </Button.Secondary>
       )}
+      {typeof deletingCategory === 'number' && (
+        <Dialog
+          header={statsText.deleteCategory()}
```

Comment:
would be good to include category name in the dialog (perhaps in the dialog header) so that it's clearer which category they are deleting - this is a good practice

see Asana:

![Screenshot 2023-07-06 at 19 49 11](https://github.com/specify/specify7/assets/40512816/0a836d3a-6f16-4c27-b053-8a54e6d2992b)

Reply:
Added category name in the dialog header.

Reply:
Hiii, could you also add the red circle with exclamation point image in the top left corner? 

Additionally, to be consistent with other warning dialogs, we may want to make the question the dialog header and add the category name to the description. 

`example`
![Screen Shot 2023-07-07 at 9 09 11 AM](https://github.com/specify/specify7/assets/135047322/ee091a3d-1204-4518-aa6b-0059bc53c4e9)